python-client/api-reference/attachment.mdx

```diff
@@ -54,11 +55,17 @@ attachment = client.api.create_attachment(
   Path in string format
 </ParamField>
 
+### Return type
+
+<ResponseField name="attachment" type="Attachment">
+  Return an Attachment
+</ResponseField>
+
 ## Update attachment
 
 ```python
 attachment = client.api.update_attachment(
-    id="1234",
+    id="c0bd9bf3-45f2-447b-ae67-d4c18ae06542",
```

Comment:
it's a bit confusing to see hard coded values here. Maybe write "ATTACHEMENT_UUID" instead?